data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+			`	With the little "tour" over, when you three reach your ship, you show them to their bunks, and prepare to head back to <planet>.`
+				accept
+
+
+
+mission "Coalition Yottrite 6"
+	name "Master Shipwright"
+	description "Head to <destination> to pick up a member of House Mallob, so that they may help in trying to apply Yottrite to an alloy."
+	landing
+	to offer
+		has "Coalition Yottrite 5: done"
+	source "Delve of Bloptab"
+	destination "Market of Gupta"
+	on offer
+		conversation
+			`Lugglop is waiting for you when you land, and greets the two researchers as you help them get their things out of the ship.`
+			`	He points them to a hotel, where he had a room prepared for them, and asks you to follow him to his office again.`
+			`	"Filled to the brim with cutting edge equipment, our facilities here and on the neighbor planet are. Alas, venture much into the theoretical field needed to study such rare elements, our personnel here does not."`
+			`	He shows you to his map again, this time pointing to a nearby Arachi world. "Once cracked down some aspects of Yottrite, the twins have, an expert in ship making, we will need," he says. "On <destination>, prepared one of their representatives, House Mallob has. Bring them here, you must."`
+			`	You're about to head back to your ship, when he hands you 200,000 credits. "Nearly forgot to pay you for this last service, I did."`
+				accept
+	on accept
+		payment 200000
+
+
+
+mission "Coalition Yottrite 7"
+	name "Master Shipwright"
+	description "Bring Pichiie to <destination>, where she will join Lugglop's team in their Yottrite research."
+	passengers 1
+	landing
+	to offer
+		has "Coalition Yottrite 6: done"
+	source "Market of Gupta"
+	destination "Delve of Bloptab"
+	on offer
+		conversation
+			`You're immediately hailed by House Mallob when your ship is descending the atmosphere of <origin>. They guide you through the planet, bringing you to what looks like a massive castle-like structure, which appears to be made out of a similar material to that of Arach ships, at the base of a mountain.`
+			`	Other buildings are around as well, and you can see other ships landing close to your own. A dedicated docking area was built here, and you're almost certain the building closest to it is something of an outfitter. The atmosphere feels like that of a small village turned into an outdoor shopping mall.`
+			`	You're greeted by an Arach, who came in what you believe to be their version of a car, though you gave up on trying to guess the exact purpose of each pipe and plating, why some wheels weren't touching the ground, while pondering if you would comfortably fit inside one.`
```

Comment:
```suggestion
			`	You're greeted by an Arach who arrived in what looks like a car (though you gave up on trying to guess the exact purpose of each pipe and plating as well as why some wheels aren't touching the ground). You wonder if you would comfortably fit inside one.`
```
Not a big fan of the current writing of this description. As above, I've suggested a fix, but I would recommend a rewriting if you decide not to use mine.

Reply:
also redid